template/razor.go

```diff
@@ -17,7 +17,11 @@ func (t *Template) applyRazor(content []byte) (result []byte, changed bool) {
 	t.ensureInit()
 
 	for _, ignoredExpr := range t.ignoredRazorExpr {
-		content = regexp.MustCompile(t.delimiters[2]+ignoredExpr).ReplaceAllFunc(content, func(match []byte) []byte {
+		if strings.HasSuffix(ignoredExpr, "*") || strings.HasSuffix(ignoredExpr, "+") {
+			ignoredExpr += `?` // Ensure that the regex is not greedy
+		}
+		ignoredExpr = t.delimiters[2] + ignoredExpr + `\b` // Stop at the end of the expression
```

Comment:
je pense que le `t.delimiters[2]` mérite un petit commentaire; j'ai du fouiller un peu pour catcher que c'est like `['{{', '}}', '@']`.

Reply:
Yes, you are right. Good catch